app/pages/about/publications-page.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import Publications from '../../lib/publications';
+import counterpart from 'counterpart';
+import Loading from '../../components/loading-indicator';
+
+export default class PublicationsPage extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      currentSort: 'showAll',
+    };
+    this.showPublicationsList = this.showPublicationsList.bind(this);
+
+  }
+
+  componentDidMount() {
+    this.loadProjects();
+  }
+
+  showPublicationsList(key) {
+    this.setState({currentSort: key});
+  }
+
+  projectSlugs() {
+    const slugs = [];
+    Object.keys(Publications).forEach((category) => {
+      Publications[category].forEach((project) => {
+        slugs.push(project.slug);
+      });
+    });
+    return slugs;
+  }
+
+  loadProjects() {
+    const slugs = this.projectSlugs();
+    const page_size = slugs.length;
+    apiClient.type('projects')
+        .get({
+          slugs,
+          cards: true,
+          page_size
+        }).then((result) => {
+          const projects = {};
+          result.forEach((project) => {
+            projects[project.slug] = project;
+          });
+          this.setState({
+            projects
+          });
+        });
+  }
+
+  renderSideBarNav(sideBarNav) {
+    return (
+      <nav>
+        { Object.keys(sideBarNav).map((navItem) => {
+            return (<button key={navItem}
+                      style={this.state.currentSort === navItem ? { fontWeight: 700 } : null }
+                      onClick={() => this.showPublicationsList(navItem)}
+                      className={`secret-button side-bar-button`}>
+                      <span>{sideBarNav[navItem]}</span>
+                    </button>);
+          })
+        }
+      </nav>
+    );
+  }
+
+  renderHeading(sideBarNav) {
+    return (
+      <h2> 
+        { this.state.currentSort === 'showAll' 
+            ? counterpart("about.publications.content.header.showAll") 
+            : sideBarNav[this.state.currentSort]
+        }
+      </h2>
+    );
+  }
+
+  renderProjects() {
+    return (
+      Object.keys(Publications).map((category) => {
+          if(this.state.currentSort == category || this.state.currentSort == 'showAll') {
```

Comment:
Please add a space after the `if` and use `===` instead of `==` in both instances.

I think the if block may be indented an extra indent?